archipelago/docs/en_Donkey Kong 64.md

```diff
@@ -29,43 +31,39 @@ Currently, two goals can be selected:
 
 ## What items and locations get shuffled?
 
-### Items
+The locations of the following items will be shuffled from their normal locations:
 
 - Playable Kongs
 - Golden Bananas
 - Banana Fairies
 - Banana Medals
 - Battle Crowns
 - All Keys for K. Lumsy's cage, except Key 8
-- Blast-o-Matic blueprints
+- Blueprints
 - Instruments, guns and respective upgrades
 - Training Grounds moves
-- Cranky's Lab moves/potions
+- Moves purchased from shops
 - The Banana Fairy Camera
```

Comment:
Shockwave as well, not just Fairy Camera. Looks good other than that. Sorry for the late minor nag.